molgenis/ds-tidyverse
# feat: bind-cols

## To test
See instructions on clientside [pull request](https://github.com/molgenis/ds-tidyverse-client/pull/48)
